src/library/scala/util/Try.scala

```diff
@@ -141,6 +141,61 @@ sealed abstract class Try[+T] extends Product with Serializable {
     def withFilter(q: T => Boolean): WithFilter = new WithFilter(x => p(x) && q(x))
   }
 
+  /** Tests whether this is a `Success` with a given value.
+   *
+   * This is equivalent to:
+   * {{{
+   * tryComputation match {
+   *   case Success(x) => x == value
+   *   case Failure(_) => false
```

Comment:
```
case Success(`value`) => true
case _                => false
```
arrow alignment is a challenge in this edit window on github...

Reply:
As far as I can see, this cannot be understood outside a kind of `def foo[T](tryComputation: Try[T], value: T): Boolean`. I'd prefer code understandable without.